bagua/distributed/launch.py

```diff
@@ -127,6 +127,14 @@ def parse_args():
     parser.add_argument(
         "--default_bucket_size", type=int, default=10 * 1024 ** 2
     )  # noqa: E501
+    parser.add_argument(
+        "--bagua-net_switch_on",
+        action="store_true",
+        default=False,
+        help="Turning on this switch will load the NCCL network"
```

Comment:
Enable Bagua-Net optimization for faster communication performance. See XXX (prepare a tutorial section and put link here) for more details.